integration/findandmodify_test.go

```diff
@@ -122,13 +126,299 @@ func TestFindAndModifyErrors(t *testing.T) {
 			},
 			altMessage: "BSON field 'findAndModify.upsert' is the wrong type 'string', expected type 'bool'",
 		},
+		"LowerCaseCommand": {
+			// go driver sends `findAndModify` in camel case, but
+			// js driver sends `findandmodify` in lower case.
```

Comment:
We will check that with JS tests. No need to make tests with a Go driver to emulate JS driver

Reply:
Cool, let's remove this test case for now :hugs: